forge-gui/res/cardsfolder/upcoming/fake_your_own_death.txt

```diff
@@ -0,0 +1,11 @@
+Name:Fake Your Own Death
+ManaCost:1 B
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature | TgtPrompt$ Select target creature | NumAtt$ +2 | SubAbility$ DBAnimate | SpellDescription$ Until end of turn, target creature gets +2/+0 and gains "When this creature dies, return it to the battlefield tapped under its owner's control and you create a Treasure token." (It's an artifact with "{T}, Sacrifice this artifact: Add one mana of any color.")
+SVar:DBAnimate:DB$ Animate | Triggers$ TrigChangeZone | sVars$ TrigChangeZone2 | Defined$ ParentTarget
+SVar:TrigChangeZone:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigChangeZone2 | TriggerController$ TriggeredCardController | TriggerDescription$ When this creature dies, return it to the battlefield tapped under its owner's control and you create a Treasure token. (It's an artifact with "{T}, Sacrifice this artifact: Add one mana of any color.")
+SVar:TrigChangeZone2:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | Tapped$ True | Defined$ TriggeredNewCardLKICopy | SubAbility$ DBToken
+SVar:DBToken:DB$ Token | TokenScript$ c_a_treasure_sac | TokenOwner$ TriggeredCardController
```

Comment:
you can probably use `You` in that context because it is an LKI trigger

also the `sVars$ TrigChangeZone2` shouldn't be needed